antenna/ext/s3/crashstorage.py

```diff
@@ -66,8 +65,7 @@ def check_health(self, state):
         self.connection.check_health(state)
 
     def _get_raw_crash_path(self, crash_id):
-        return "v2/raw_crash/{entropy}/{date}/{crash_id}".format(
-            entropy=crash_id[:3],
+        return "v2/raw_crash/{date}/{crash_id}".format(
```

Comment:
I just re-re-re-read the bug and noticed I intended to do a `v3`. But this is really just like v1, so we should just switch it back. Nothing else depends on the versioning of the file system.